sql/migrations/20200516095255_world.sql

```diff
@@ -0,0 +1,55 @@
+DROP PROCEDURE IF EXISTS add_migration;
+delimiter ??
+CREATE PROCEDURE `add_migration`()
+BEGIN
+DECLARE v INT DEFAULT 1;
+SET v = (SELECT COUNT(*) FROM `migrations` WHERE `id`='20200516095255');
+IF v=0 THEN
+INSERT INTO `migrations` VALUES ('20200516095255');
+-- Add your query below.
+
+
+-- add correct gossip_menu_entries for alchemist narett
+UPDATE `gossip_menu` SET `condition_id` = 362 WHERE `entry` = 4122 AND `text_id` = 5050;
+INSERT INTO `gossip_menu` (`entry`, `text_id`, `condition_id`) VALUES
+(4122, 5051, 25),
+(4122, 5053, 361);
+
+
+-- add correct gossip_menu_entries for lilyssia nightbreeze (I believe this is correct as one of the broadcasts mentions the npc next to her)
+DELETE FROM `gossip_menu` WHERE `text_id` = 5016 AND `entry` = 4201;
+INSERT INTO `gossip_menu` (`entry`, `text_id`, `condition_id`) VALUES
+(4201, 5316, 0),

```

Comment:
This text seem to be for when player has too low skill to train with her. Should have condition.

```
My teachings would be too advanced for you at this stage. If you wish to learn more of potions, speak with Tel'Athir here instead.
```